adyenv6core/resources/adyenv6core-items.xml

```diff
@@ -202,6 +202,34 @@
                     <modifiers optional="false"/>
                     <defaultvalue>Boolean.FALSE</defaultvalue>
                 </attribute>
+                <attribute qualifier="adyenPOSEnabled" type="java.lang.boolean">
+                    <description>Enable POS Transactions</description>
+                    <persistence type="property"/>
+                    <modifiers optional="false"/>
+                    <defaultvalue>Boolean.FALSE</defaultvalue>
+                </attribute>
+                <attribute qualifier="adyenPOSMerchantAccount" type="java.lang.String">
+                    <description>POS Merchant Account</description>
+                    <persistence type="property"/>
+                    <modifiers optional="true"/>
+                </attribute>
+                <attribute qualifier="adyenPOSApiKey" type="java.lang.String">
+                    <description>POS API Key</description>
+                    <persistence type="property"/>
+                    <modifiers encrypted="true" optional="true"/>
+                </attribute>
+                <attribute qualifier="adyenPOSStoreId" type="java.lang.String">
```

Comment:
It is Id here

Reply:
I see the adyenPOSStoreId for qualifier here. Do you mean the description ?<description>POS Store ID</description> to <description>POS Store Id</description>

Reply:
Yes I mean it is not consistent. So this is correct one.(adyenPOSStoreId)

Reply:
The qualifier is adyenPOSStoreId so it is correct.